config/manager/kustomization.yaml

```diff
@@ -9,5 +9,5 @@ generatorOptions:
 
 images:
 - name: controller
-  newName: quay.io/jhernand/o2ims-operator
-  newTag: "2"
+  newName: quay.io/imihai/oran-o2ims-operator
```

Comment:
Could we change this to the final name which would be `quay.io/openshift-kni/oran-o2ims-operator`?

Reply:
Yes, will do.